client/profile-wizard/steps/business-details.js

```diff
@@ -392,6 +444,132 @@ class BusinessDetails extends Component {
 		);
 	}
 
+	renderBusinessExtensionsBundle( values, getInputProps ) {
+		const { isPopoverVisible } = this.state;
+
+		return (
+			<div className="woocommerce-business-extensions">
+				<label htmlFor="woocommerce-business-extensions__checkbox">
+					<CheckboxControl
+						id="woocommerce-business-extensions__checkbox"
+						{ ...getInputProps( 'install_extensions' ) }
+					/>
+					<span className="woocommerce-business-extensions__label-text">
+						{ __(
+							'Install recommended free business features',
```

Comment:
[In the design](https://user-images.githubusercontent.com/63922/85745425-b046b500-b6d3-11ea-8c34-a5a419a735eb.png), it looks like the word `free` is in bold

Reply:
Good eyes!  Updated in https://github.com/woocommerce/woocommerce-admin/pull/4695/commits/10057953372d071373b129050c6d44a50fb0df1a